### PR TITLE
Debian 11 support and updated dependencies

### PR DESCRIPTION
**Pull request by antonioarenzanapulido**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change
Updated dependencies on test-requirements.txt and added Debian 11.

### Benefits

Updated dependencies and debian 11 support.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/nexus-role/tree/develop)
```
Comment (ablancolopez) [inline CHANGELOG.md:6]: when updating master, you should also update the changelog in order to reflect the version you are going to release :)

File: meta/main.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 galaxy_info:
+  role_name: nexus_role
+  namespace: idealista
   company: Idealista S.A.
```
Comment (ablancolopez) [inline meta/main.yml]: The company name is Idealista S.A.U.

Could you also reflect here that the role supports Debian 11 in the `platforms` section?

File: CHANGELOG.md (hunk 1 above)
Comment (antonioarenzanapulido) [reply CHANGELOG.md:6]: Fixed :)

File: meta/main.yml (hunk 2 above)
Comment (antonioarenzanapulido) [reply meta/main.yml]: Fixed :)

**Pull request merged**